test/verified_test.py

```diff
@@ -351,4 +352,8 @@
 True
 >>> int(convert_water(240, "pt", True)) == 113562 # https://www.inchcalculator.com/convert/milliliter-to-pint/
 True
+>>> math.trunc(convert_water(240, "qt") * 10**6) / 10**6 == 0.253605 # https://www.inchcalculator.com/convert/volume/
```

Comment:
There is no need to use `math.trunc`; it's not crucial to get exact website number (due to precision), but rather to be almost equal.